tests/tests_suite_test.go

```diff
@@ -90,10 +91,14 @@ var _ = BeforeSuite(func() {
 			types.WithID(machineID),
 			types.WithSSHUser(user()),
 			types.WithSSHPass(pass()),
+			types.WithMemory("6000M"),
```

Comment:
argh, ok, this is not compatible with vbox apparently. vbox doesn't want the `M` :(